src/manifest.json

```diff
@@ -26,7 +26,7 @@
     "*://*/*"
   ],
 
-  "content_security_policy": "script-src 'self'; object-src 'self'",
```

Comment:
reason:
https://bugzilla.mozilla.org/show_bug.cgi?id=1294996#c5

https://getalby.slack.com/archives/C02TD6P5XPE/p1671564553376349?thread_ts=1671514296.425669&cid=C02TD6P5XPE

Reply:
👏 great find @im-adithya :D

Reply:
What are the security implications of changing the CSP? Now an attacker could execute any code that he manages to get into a a blob and add to the page?